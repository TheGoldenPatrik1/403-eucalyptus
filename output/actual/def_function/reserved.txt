Error on line 1: Cannot define Function with reserved name 'print'
