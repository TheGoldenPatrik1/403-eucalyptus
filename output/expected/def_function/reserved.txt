Error on line 1: Cannot define function with reserved name 'print'